projects/ad9172/src/main.c

```diff
@@ -291,6 +296,32 @@ int main(void)
 	axi_dac_set_datasel(tx_dac, -1, AXI_DAC_DATA_SEL_DDS);
 #endif /* DAC_DMA_EXAMPLE */
 
+#ifdef IIO_SUPPORT
+	printf("The board accepts libiio clients connections through the serial backend.\n");
```

Comment:
I think this message should be put under iio_server_init, otherwise we duplicate it in each project.

Reply:
Indeed. But either it is duplicated here or duplicated in the app_iio.c file, since every project has it's own `iio_server_init` function (the number of function parameters and the type may vary from project to project).